tests/www/views/test_session.py

```diff
@@ -95,10 +95,4 @@ def test_check_active_user(app, user_client):
     user.active = False
     resp = user_client.get("/home")
     assert resp.status_code == 302
-    assert "/login" in resp.headers.get("Location")
-
-    # And they were logged out
```

Comment:
I removed this test because now the actual logout operation is done within Flask itself. But testing that the user is redirected to `/logout` is good enough

Reply:
Do we have (need?) a test for logout redirecting to the correct page?

Reply:
I dont think we need it because we get the logout URL from `f"{self.security_manager.auth_view.endpoint}.logout"`  that is defined [here](https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/security/views.py#L499). It is handled by Flask itself so I dont think we need to test that